mindmeld/components/dialogue.py

```diff
@@ -705,7 +706,7 @@ def _get_dialogue_handler(self, dialogue_state):
         return handler
 
 
-class AutoEntityFilling:
+class AutoEntityFilling:   # pylint: disable=R0902
```

Comment:
We can avoid this lint issue by grouping together the attributes (form & local form, etc)

Reply:
I looked into that, but even with comments, it might get confusing to reuse/group attributes while maintaining it at a later stage.

Reply:
Ignore the above. The simplified logic for invoke gets rid of the extra invoke attribute.